org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ProjectDescriptor.xtend

```diff
@@ -174,12 +174,13 @@ abstract class ProjectDescriptor {
 		config.sourceLayout.getPathFor(outlet);
 	}
 
-	def outputFolder(Outlet outlet) {
-		config.sourceLayout.getOutputFor(outlet);
-	}
-
 	def isTest(Outlet outlet) {
-		config.sourceLayout.isTest(outlet);
+		for (o : Outlet.testOutlets) {
```

Comment:
looks like we should do a contains() on the testOutlets. If it is an Array, Arrays.asLists() should do the trick.

Reply:
Changed as requested.